packages/adapter-cloudflare/files/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
I'd prefer to leave this file at the root of the project where it was before

Reply:
Are you sure? The content will be ugly 
```
/files/*
!/files/.assetsignore
```
It is not a big deal to me, but it makes sense to keep `.gitignore` in the relevant folder close to the content mentioned.

Reply:
Yeah, because everywhere else we have it at the project root so it'd be really hard to remember that it's different for just this one

Reply:
Done